env-check.sh

```diff
@@ -67,6 +67,23 @@ else
     printf "$out_str"
 fi
 
+# octave packages are required only for check-volume-levles.sh
+# Good to check upfront but this can be optional requirement
+out_str="" check_res=0
```

Comment:
Missing semi-colon, empty `out_str`: that does not seem right.

Reply:
You mean `out_str=""; check_res=0` but it worked fine and no shellcheck error/warning for this.

any harm empty `out_str` by assigning ""?

Reply:
My bad, `out_str="" check_res=0` is perfectly correct without a semi-colon: https://www.gnu.org/software/bash/manual/html_node/Simple-Command-Expansion.html

`out_str` is already defined at the top of the script, redefining it here is confusing because this is not done for any other check. I think this is the contract: if a function sets `check_res` then it MUST set `out_str` too.